pkgs/development/python-modules/cvxpy/default.nix

```diff
@@ -33,6 +34,7 @@ buildPythonPackage rec {
     osqp
     scipy
     scs
+    cylp
```

Comment:
Nitpick, but ``cylp`` should be in this list in alphabetical order (and in the arguments list)